Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/transform/AuditTransforms.java

```diff
@@ -228,13 +240,13 @@ protected EventIdentificationType getEventIdentificationType(CodedValueType even
 
         // Set the Event Action Time
         try {
-            GregorianCalendar today = new java.util.GregorianCalendar(TimeZone.getTimeZone("GMT"));
+            GregorianCalendar today = new GregorianCalendar(TimeZone.getTimeZone("GMT"));
             DatatypeFactory factory = javax.xml.datatype.DatatypeFactory.newInstance();
             XMLGregorianCalendar calendar = factory.newXMLGregorianCalendar(
-                today.get(java.util.GregorianCalendar.YEAR), today.get(java.util.GregorianCalendar.MONTH) + 1,
-                today.get(java.util.GregorianCalendar.DAY_OF_MONTH), today.get(java.util.GregorianCalendar.HOUR_OF_DAY),
-                today.get(java.util.GregorianCalendar.MINUTE), today.get(java.util.GregorianCalendar.SECOND),
-                today.get(java.util.GregorianCalendar.MILLISECOND), 0);
+                today.get(GregorianCalendar.YEAR), today.get(GregorianCalendar.MONTH) + 1,
+                today.get(GregorianCalendar.DAY_OF_MONTH), today.get(GregorianCalendar.HOUR_OF_DAY),
+                today.get(GregorianCalendar.MINUTE), today.get(GregorianCalendar.SECOND),
+                today.get(GregorianCalendar.MILLISECOND), 0);
```

Comment:
Date conversion can be moved to a method in helper class if a method is not created already